harbour-fernschreiber.pro

```diff
@@ -104,3 +104,5 @@ HEADERS += \
     src/tdlibsecrets.h \
     src/tdlibwrapper.h
 
+OTHER_FILES += \
+    rpm/$${TARGET}.spec
```

Comment:
Any particular reason why to add OTHER_FILES and only to add the spec file here?

Reply:
If I started adding other files, then I would have to add all other files too, including the .yaml thing which I think is useless. Adding two out of three files would look strange, and what if you want to keep the .yaml file? So I just added the file which I edited ))

Reply:
But why is the OTHER_FILES section needed at all? All elements are listed in DISTFILES, so what's the point of OTHER_FILES?

Reply:
Just for convenience of editing, I don't think it has any other meaning

Reply:
:D Well, in this case I might consider deleting this one again. Let's see!

Reply:
Well, it's up to you. Spec isn't being edited that often. I personally prefer to have all files included into the project and accessible directly from QtCreator but it's a matter if taste and habits of course.